cmd/configmaps.go

```diff
@@ -0,0 +1,100 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/appuio/seiso/cfg"
+	"github.com/appuio/seiso/pkg/cleanup"
+	"github.com/appuio/seiso/pkg/openshift"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	core "k8s.io/client-go/kubernetes/typed/core/v1"
+)
+
+const (
+	configMapCommandLongDescription = `Sometimes ConfigMaps are left unused in the Kubernetes cluster.
+This command deletes ConfigMaps that are not being used anymore.`
+)
+
+var configMapLog *log.Entry
+
+var (
+	// configMapCmd represents a cobra command to clean up unused ConfigMaps
+	configMapCmd = &cobra.Command{
+		Use:          "configmaps [PROJECT]",
+		Short:        "Cleans up your unused ConfigMaps in the Kubernetes cluster",
+		Long:         configMapCommandLongDescription,
+		Aliases:      []string{"configmap"},
```

Comment:
Add "cm" and make commands non-case-sensitive, as Simon mentioned in the Daily.

Reply:
As we discussed the topic regarding case-sensitive we found out it's not the case with `kubectl` (the commands are case sensitive meanwhile the _objects_ aren't). Debugging and researching Cobra we couldn't find any option to disable case sensitivity.